Dockerfile

```diff
@@ -1,5 +1,16 @@
-FROM gcr.io/distroless/static:debug-nonroot
-WORKDIR /
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12 AS builder
 
-COPY manager manager
-EXPOSE 8080
+WORKDIR /build
+COPY . .
+RUN make build
+
+FROM registry.ci.openshift.org/ocp/4.12:base
+
+COPY --from=builder /build/bin/manager /bin
+USER 1001
+
+LABEL io.k8s.display-name="OpenShift Platform Operator Manager" \
+      io.k8s.description="This is a component of OpenShift Container Platform and manages the lifecycle of platform operators." \
+      io.openshift.tags="openshift" \
+      summary="This is a component of OpenShift Container Platform and manages the lifecycle of platform operators." \
+      maintainer="Odin Team <aos-odin@redhat.com>"
```

Comment:
nit: can we update this to a more recent email? We could use the olm dev working group email. 

That email is set up from Evan and I'm not sure the current status of who is monitoring it.

Reply:
Yep, just a copy/paste error. We'll have this issue across multiple repos, so probably best to update it uniformly.